code/game/events/naturify.dm

```diff
@@ -277,6 +279,21 @@ var/list/seedbush_spawns = list(
 	for(var/turf/T in target)
 		uncharge_all_cells_recursive(T)
 
+	for(var/obj/structure/sink/S in target)
+		if(!istype(S, /obj/structure/sink/puddle))
+			new /obj/structure/sink/puddle(S.loc)
+			qdel(S)
+
+	for(var/obj/item/weapon/reagent_containers/glass/bucket/buckets in target)
+		if(!istype(buckets, /obj/item/weapon/reagent_containers/glass/bucket/wooden))
+			new /obj/item/weapon/reagent_containers/glass/bucket/wooden(buckets.loc)
```

Comment:
this will delete any reagents that are in the bucket.
(i don't think any stations spawn with (notable) reagents in buckets at roundstart, so it probably won't cause issues).

Reply:
No station map starts with any reagents in any bucket. There are water_filled buckets, but those are only spawned in vaults/hoboshacks/lamprey - no standard map. The greenify proc also adds in plenty of /sink/puddles for them to fill said bucket with in the case that a vault is affected by this.